build_tools/mako/benchmark_modules_on_android.sh

```diff
@@ -47,7 +47,7 @@ while [[ $# -gt 0 ]]; do
 done
 
 TS="$(date +%s)000"
-SED_EXPR='s/BM_[[:alnum:][:punct:]]+ [[:space:]]+ ([0-9]+) ms[[:print:]]+/\1/p'
+SED_EXPR='s/BM_[[:alnum:][:punct:]]+\/real_time[[:space:]]+ ([.0-9]+) ms[[:print:]]+/\1/p'
```

Comment:
This is fine for now, but note that Benchmark has structured output as well, which is probably what we should be using (https://github.com/google/benchmark#output-formats)

Reply:
Got you, I just feel this is easy enough, instead of parsing a json file in bash. CSV format is probably good.